misc/scripts/mockgen.sh

```diff
@@ -102,6 +95,13 @@ mockgen -destination=./test/mock/mock_chainmanager/mock_chainmanager.go  \
         -package=mock_chainmanager \
         StateManager
 
+mkdir -p ./test/mock/mock_blocksync
+mockgen -destination=./test/mock/mock_blocksync/mock_blocksync.go  \
+        -source=./blocksync/blocksync.go \
+        -self_package=github.com/iotexproject/iotex-core/blocksync \
+        -package=mock_blocksync \
+        BlockSync
+
```

Comment:
make mockgen complains about error in block_sync
move it here will get rid of those error